### PR TITLE
[ADD][8.0] stock operation type location

### PR DESCRIPTION
**Pull request by mikevhe18**
In case to avoid user on selecting a wrong location on stock operation, i have made this module which has a function to filtering location  and destination location.

### CODE REVIEW

**Comment by andhit-r** [comment]
Functionality tested on runbot :+1:

**Comment by mikevhe18** [comment]
Hello @gurneyalex , @tafaRU , @jgrandguillaume.
Would you kindly review this PR again ?

Thanks in Advance

**Comment by gurneyalex** [comment]
:+1:

**Comment by rousseldenis** [comment]
:+1:  Tested on runbot
Remark : Would this behaviour be extended to Transfer Wizard ?

**Comment by mikevhe18** [comment]
@rousseldenis Thank you for review. Noted, your request has been put in my to-do list

File: stock_operation_type_location/README.rst [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    
+=============================
+Stock Operation Type Location
+=============================
+
+This module was written to extend the functionality of the stock operations 
+which allows users to filter destination and source locations when adding products.
+
+Usage
+=====
```
Comment (andhit-r) [inline stock_operation_type_location/README.rst]: I think this section goes bellow installation section

File: stock_operation_type_location/README.rst [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+To install this module, you need to:
+
+1.  Clone the branch 8.0 of the repository https://github.com/OCA/stock-logistics-warehouse
+2.  Add the path to this repository in your configuration (addons-path)
+3.  Update the module list
+4.  Go to menu *Setting -> Modules -> Local Modules*
+5.  Search For *Stock Operation Type Location*
+6.  Install the module
+
+Usage
```
Comment (andhit-r) [inline stock_operation_type_location/README.rst:25]: There are already usage section above

File: stock_operation_type_location/__init__.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# © 2016 OpenSynergy Indonesia
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from . import models
+from . import tests
```
Comment (andhit-r) [inline stock_operation_type_location/__init__.py]: You do not import tests, it's automatically execute

**Review by andhit-r — CHANGES_REQUESTED** [review]
Minor comment

**Pull request merged**